geometry_msgs/PoseStamped pose_msg
string additional_info
